docs/basics/Gotchas/Gotchas.md

```diff
@@ -0,0 +1,7 @@
+# Gotchas
+Every distro is going to have it's own set of rules and common problems.  It's highly recommended you familiarize your
```

Comment:
Looks like you've got a lot of double-spaces in here, in inconsistent places. Would be good to strip those out (back to 1 space, I mean).